app/models/rules_factory.rb

```diff
@@ -4,10 +4,18 @@ def initialize(auction)
   end
 
   def create
+    eligibility = if auction.small_business?
```

Comment:
I've seen `Pool` in the Agile BPA context and I don't think that's what we want here. A pool would be more of a vetted/pre-approved list of vendors for a specific purpose.

FWIW I really kind of do like eligibility. It sort of fits well with the error messages we would display to a user:

```
Only small businesses are eligible to bid on this auction.
```

What are the issues you see in it, @harrisj?